docs/src/pages/docs/installation/email_reports.mdx

```diff
@@ -6,8 +6,309 @@ index: 10
 version: 1
 ---
 
-## Scheduling and Emailing Reports
+## Alerts and Reports
```

Comment:
I'm not really comfy about putting the Alerts & Reports doc in a page named `email-reports`, as this doc concern alerts too, and not only emails but also Slack :shrug:

Reply:
I'd definitely agree, was really just for organisation so that in the menu there isn't 'Alerts and reporting' and then below it 'Scheduling and Emailing reports' which would be confusing imo, but if @srinify can help we could put it into its own section 'Alerts and Reporting', and then have the legacy reporting stuff remain at the bottom of this file?

Reply:
I agree, renaming and keep the old approach at the end seems the best!